.github/workflows/build.yml

```diff
@@ -29,6 +29,14 @@ jobs:
           cmake -B cmake-build -D CRASHPAD_BUILD_TOOLS=On
           cmake --build cmake-build --parallel
 
+      - name: Build crashpad Windows ARM64
+        if: ${{ runner.os == 'Windows' }}
+        run: |
+          cmake --version
+          & "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvarsamd64_arm64.bat"
```

Comment:
will this be stable across time and updates to the runner image?

Reply:
That won't be in non-draft PR. I only added this to check for the setup it shouldn't be necessary to set up the CLI for the MSBuild generator.

Reply:
I would recommend that you take a look at [compnerd/gha-setup-vsdevenv](https://github.com/compnerd/gha-setup-vsdevenv)

Reply:
Thanks. This could be helpful in some situations. But as noted above, for the given build, you don't need to prep vcvars except for debugging. If configured correctly and with unique build options, the generator should be able to pick up all required tools, even for cross-compilation.